src/components/organisms/EventModal/EventModal.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useMemo } from "react";
+import Modal from "react-bootstrap/Modal";
+import { VenueEvent } from "types/venues";
+import { Room } from "types/rooms";
+
+import { WithVenueId } from "utils/id";
+import { enterVenue } from "utils/url";
+import { hasEventFinished, isEventLive } from "utils/event";
+import { formatUtcSecondsRelativeToNow } from "utils/time";
+
+import { useRelatedVenues } from "hooks/useRelatedVenues";
+import { useRoom } from "hooks/useRoom";
+
+import { Button } from "components/atoms/Button";
+
+import "./EventModal.scss";
+
+const emptyRoom: Room = {
```

Comment:
hmm might be worth moving this constant somewhere else? :thinking:

Reply:
removed as discussed here: https://github.com/sparkletown/sparkle/pull/1426#discussion_r637907869